exercises/complex-numbers/canonical-data.json

```diff
@@ -68,7 +68,8 @@
     {
       "description": "Imaginary unit",
       "property": "mul",
```

Comment:
now I'm thinking... surely `i * i = -1` would be an easier test than `(1+2i) * (3+4i) = -5 + 10i` right? any chance it should get moved before?